tests/cypress/support/commands.js

```diff
@@ -802,6 +829,14 @@ Cypress.Commands.add('exportTask', ({
     cy.closeNotification('.cvat-notification-notice-export-task-start');
 });
 
+Cypress.Commands.add('renameTask', (oldName, newName) => {
+    cy.get('.cvat-task-details-task-name').within(() => {
+        cy.get('[aria-label="edit"]').click();
+    });
+    cy.contains('.cvat-text-color', oldName).clear().type(`${newName}{Enter}`);
+    cy.contains('.cvat-task-details-task-name', newName).should('exist');
+});
+
```

Comment:
Hi, @dvkruchinin 

Text for ellipses has been added here #4229

Reply:
Hi @bsekachev,

Great!
I`ll update the cypress command implemented in #4215